README.md

```diff
@@ -176,4 +176,6 @@ browserWindow.LastName.Text ="Doe";
 
 // Click on the Save button
 browserWindow.Save.Click();
-```
\ No newline at end of file
+```
+
+Follow the steps from [this guide](http://www.codewrecks.com/blog/index.php/2011/06/06/witch-version-of-browser-is-used-by-the-webbrowser-control/) to use a more recent version of IE in the Object Recorder.
```

Comment:
Thank you for the clarification; that makes more sense now.
I'm thinking we can make things easier by actually updating the Object Recorder to dynamically set the version of IE to use during recording.